scripts/generate-git-snapshot

```diff
@@ -234,7 +234,7 @@ identify_latest_change() {
   fi
 
   echo "Latest tag [${last_tag:-}] / merge [${last_merge:-}] seems to be $since"
-  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS
+  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS $DCH_CHANGELOG_FILE
```

Comment:
For example if you look into the Kamailio repository at https://github.com/kamailio/kamailio, you see that there is no debian folder in the project root. The debian folder is located under pkg/kamailio/deb/$distribution. I do symlink this folder to ./debian before starting the build, but when git-dch inspects ./debian/changelog, it finds no commit for the latest changelog entry. So I had to point git-dch to the changelog file where it is actually committed. I put the line "DCH_CHANGELOG_FILE=pkg/kamailio/deb/$distribution/changelog" into my project configuration. Otherwise git-dch will start building a changelog entry for all commits that exist in the project, but that will run forever.
I've seen other projects too, where the debian folder is located in some subfolder of the project, so I think this could help in other cases as well.

I'll change the syntax, thanks for the hint. :)